### ISSUE TITLE
Running `helm diff` will always yield a change in the `memberlist` secret

### ISSUE THREAD

**Issue by phihos** [bitnami/charts#5620]
**Which chart**:
metallb 2.3.2

**Describe the bug**

Running `helm diff` will always yield a change in the `memberlist` secret. The is because the existing `speaker.secretValue` is unquoted and the "new" (but same) `speaker.secretValue` is quoted. 
I already traced the behavior down to `common.secrets.passwords.manage` but I could not find the actual culprit.

**To Reproduce**

1. Create a `values.yaml` with `speaker.secretValue` set to a fixed value.
2. Deploy with helm for the first time: `helm install --values values.yaml metallb bitnami/metallb`
3. Install helm-diff plugin and run something like `helm diff upgrade --values values.yaml metallb bitnami/metallb --show-secrets`

**Expected behavior**

The diff should be empty.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:17:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.9+k3s1", GitCommit:"630bebf94b9dce6b8cd3d402644ed023b3af8f90", GitTreeState:"clean", BuildDate:"2020-09-17T19:05:07Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by juan131** [comment]
Hi @phihos 

It's easy to reproduce by simply running this:

```console
$ helm template metallb bitnami/metallb -s templates/speaker/secret.yaml > secret.yaml
$ yq read secret.yaml
# Source: metallb/templates/speaker/secret.yaml
apiVersion: v1
data:
  secretkey: "c29tZS12YWx1ZQ=="
...
$ kubectl create -f secret.yaml
$ kubectl get secret metallb-memberlist -o yaml
apiVersion: v1
data:
  secretkey: c29tZS12YWx1ZQ==
kind: Secret
...
```

This comes from:

- https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_secrets.tpl#L90

That final: `| quote }}` is adding the quotes that are provoking the `helm-diff` plugin to fail. 

Honestly, I'm not sure if this is really a problem in the chart or a limitation in the plugin that's unable to recognise equivalent YAML files.

**Comment by phihos** [comment]
Yes you are right. Probably this change causes far more trouble than fixing the helm diff plugin.

**Comment by juan131** [comment]
@javsalgar what do you think about this?
